pkg/controller/certificates/certificate_request.go

```diff
@@ -767,6 +767,7 @@ func (c *certificateRequestManager) buildCertificateRequest(log logr.Logger, crt
 			Annotations: map[string]string{
 				cmapi.CRPrivateKeyAnnotationKey: crt.Spec.SecretName,
 			},
+			Labels: crt.Labels,
```

Comment:
Not copying this field across caused label selectors to not work when using the new CertificateRequest implementation.

I've got a WIP conformance suite that will catch this sort of thing, that I will follow up with later today 😄